.github/workflows/main.yaml

```diff
@@ -64,3 +66,4 @@ jobs:
       job-environment: "org-sagebase-strides-ampad-workflows"
       sceptre-suffix: "ampad"
       tower-url: "https://tower.sagebionetworks.org"
+      aws-role-to-assume: "arn:aws:iam::751556145034:role/sagebase-github-oidc-nextflow-infra"
```

Comment:
the role name is generated by AWS, will need deploy then get the name and put it here.

Reply:
@zaro0508 is this done?